snare/utils/timeout_parser.py

```diff
@@ -0,0 +1,16 @@
+def parse_timeout(timeout):
```

Comment:
you can combine different helper functions in one file, no need to make separate file for each function